core/jazz_is-slack-channel-available/components/config.js

```diff
@@ -0,0 +1,46 @@
+// =========================================================================
```

Comment:
you will have to use the new version
https://github.com/tmobile/jazz/blob/v1.7.1/core/jazz_usermanagement/components/config.js

Reply:
Included new version config file